newtab.js

```diff
@@ -907,13 +916,15 @@ function toggle(node, a) {
 		// open folder
 		if (a.nextSibling)
 			animate(node, a, isopen);
-		else
+		else{
+			delete node.children;

```

Comment:
We need to delete children so `getChildrenFunction` loads the new sorted children.